scene/main/scene_tree.cpp

```diff
@@ -1075,11 +1076,13 @@ Ref<ArrayMesh> SceneTree::get_debug_contact_mesh() {
 	/* clang-format on */
 
 	Vector<int> indices;
+	indices.reserve(8 * 3);
 	for (int i = 0; i < 8 * 3; i++) {
 		indices.push_back(diamond_faces[i]);
 	}
 
 	Vector<Vector3> vertices;
+	vertices.reserve(6);
```

Comment:
Should the paradigm here should be:
* resize without initializing
* set each individually